### PR TITLE
Add hover in dark mode and delete unnecessary css

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket

https://trello.com/c/kUYUN01m/266-qc-round-v-100-r

# What solved
✅Add hover in dark mode and delete unnecessary css

# Actions
Add hover in dark mode and delete unnecessary css

### CODE REVIEW

File: src/stories/containers/RecognizedDelegates/components/FilterDelegate/DelegateSelectItem.tsx [hunk 1]
```diff
@@ -80,9 +80,21 @@ const CheckWrapper = styled.span({
...
+          : !isDisable && checked
+          ? '#EDEFFF'
+          : '#F6F8F9'
+        : isDisable && checked
+        ? '#231536'
+        : isDisable && !checked
+        ? 'none'
+        : !isDisable && checked
+        ? '#231536'
+        : '#25273D',
```
Comment (YasielCabrera) [inline src/stories/containers/RecognizedDelegates/components/FilterDelegate/DelegateSelectItem.tsx:97]: As this has 3 different variables, it is pretty hard to follow, would be nice to create a variable in the styled component function to make it more readable. Maybe something like this:
```ts
const StyledContainer = styled(Container)<{ isLight: boolean; checked: boolean; isDisable: boolean }>(
  ({ isDisable, checked, isLight }) => {
  let background: string;
  if (isLight) {
    if (isDisabled) {
      background = checked ? 'xxx' : 'yyy';
    } else {
      background = checked ? 'qqq' : 'www';
    }
  } else {
    // ...
  }

  return ({
    '&:hover': {
      background,
    }
  })
});
```

Comment (alejandrocabriales) [reply src/stories/containers/RecognizedDelegates/components/FilterDelegate/DelegateSelectItem.tsx]: I got your point

**Review by YasielCabrera — APPROVED** [review]
LGTM
